src/layer/leaflet.motion.seq.js

```diff
@@ -73,19 +79,41 @@ L.Motion.Seq = L.Motion.Group.extend ({
 		return allLayers.length ? allLayers[0] : null;
 	},
 
+	addLayer: function (l) {
+		this.__prepareLayer(l);
+		L.Motion.Group.prototype.addLayer.call(this, l);
+		// If we have started animating but don't have a current layer,
+		// start this one.
+		if (this._started && !this._activeLayer) {
+			l.motionStart();
```

Comment:
If we created a SEQ with few layers, have not started it, and add one more layer on top of it, it will start running from this last layer.